src/Umbraco.Web.Common/Security/ConfigureIISServerOptions.cs

```diff
@@ -1,18 +1,24 @@
+﻿using System;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.Extensions.Options;
 using Umbraco.Cms.Core.Configuration.Models;
 
 namespace Umbraco.Cms.Web.Common.Security
 {
-    public class ConfigureIISServerOptions : IConfigureOptions<IISServerOptions>
+    [Obsolete("This class is obsolete, as this does not configure your Maximum request length")]
```

Comment:
It might make sense to wait with merging this until the documentation is merged so we can add the link immediately?

Reply:
Yea i think that would probably be best 🙈